beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -315,7 +316,12 @@ func (vs *Server) broadcastAndReceiveBlobs(ctx context.Context, sidecars []*ethp
 		if err != nil {
 			return errors.Wrap(err, "ROBlob creation failed")
 		}
-		if err := vs.BlobReceiver.ReceiveBlob(ctx, blocks.NewVerifiedROBlob(readOnlySc)); err != nil {
+		verifiedBlob := blocks.NewVerifiedROBlob(readOnlySc)
+		vs.OperationNotifier.OperationFeed().Send(&feed.Event{
```

Comment:
should this be under the operations feed or should I create a new notifier for blobs or use the block notifier instead?

Reply:
operation feed makes sense...as it is for all objects that go inside of blocks

Reply:
The other place we fire this event is in `blobSubscriber`. `blobSubscriber` and this function (line 324) both call `ReceiveBlob`. `ReceiveBlob` is responsible for actually writing the blob to disk and pushing it to a possibly blocked `ReceiveBlock` call. In terms of timing, I think it makes sense to publish the blob on the event feed after it has been saved and to prioritize the da check over the event feed, so I would suggest moving the event firing to the end of `ReceiveBlob` which ensures the timing is the same in both cases.

Reply:
>  should I create a new notifier for blobs or use the block notifier instead
I need more context - why do you want to move it to a different notifier?

Reply:
re moving the event firing to `ReceiveBlob`, heard from James that this requires more extensive plumbing to get the notifier into the blockchain package, so I will approve this as-is.